libp2p/ENR.cpp

```diff
@@ -279,5 +280,12 @@ std::ostream& operator<<(std::ostream& _out, ENR const& _enr)
     return _out;
 }
 
+std::string ENR::textEncoding() const
+{
+    RLPStream s;
+    this->streamRLP(s);
+    return toBase64URLSafe(std::string(s.out().begin(), s.out().end()));
```

Comment:
I think something like 
```suggestion
    return toBase64URLSafe(ref(s.out()));
```
should work